dashboard/app/views/pd/application/teacher_application/logged_out.haml

```diff
@@ -1,25 +1,51 @@
 - content_for(:head) do
   = stylesheet_link_tag 'css/pd', media: 'all'
 
+:css
+  /* Make the X in regional partner mini-contact form's header look correct. */
+  div.modal-content .modal-header .close {
+    height: 34px
+  }
+
 %h1
-  = I18n.t('pd.logged_out.heading')
+  Start your application
+
+.teacher-application-logged-out
+  .paragraph
+    To get started, you need to be logged into a Code.org account.
+
+  .paragraph
+    If you’d like more information about the program before you start your application, please check out the
+    = link_to("Professional Learning Program overview", CDO.code_org_url('/educate/professional-learning/middle-high'), target: "_blank")
+    or
+    = succeed "." do
+      #regional-partner-mini-contact-popup-link-container{"data-source-page-id" => "teacher-application-logged-out", "data-options" => {notes: "Please tell me more about the professional learning program!"}, "data-link-text" => "contact your local regional partner"}
 
-%p
-  != I18n.t('pd.logged_out.body', url: CDO.code_org_url('/educate/professional-learning/middle-high'))
```

Comment:
We’re losing some translations here- should these new strings be translated?

Reply:
I checked with @boatmarina and this page, as part of the teacher application, is en only, so I believe this is okay.